### DISCUSSION TITLE
AOC Day 12 Discussion - Climb Every Mountain

### DISCUSSION THREAD

**Discussion by rhdeck** [Virtual-Coffee/virtualcoffee.io#772] [Advent of Code 2022]
Discussion of solutions for [Advent of Code Day 12](https://adventofcode.com/2022/day/12) here.

Don't forget about the VC Advent of Code private leaderboard:

Go to [leaderboard/private](https://adventofcode.com/2022/leaderboard/private)
Use join code 363496-e6f42283

**Comment by rhdeck** [comment]
I never really learned Djikstra, so I used this one try to rediscover it naively. Not sure it's totally pretty, but I'm proud of the [solve](https://github.com/rhdeck/aoc-2022/blob/main/src/12/day_12.ipynb).

**Comment by doleraj** [comment]
I think I implemented A* at least 3 times today before I got it right. I am very dumb and sad today. https://github.com/doleraj/adventofcode/blob/main/aoc2022/src/day12/index.ts
